sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -131,7 +107,9 @@ private AnalyzeSentimentResult convertToAnalyzeSentimentResult(final DocumentSen
                 new RuntimeException(String.format(Locale.ROOT, "'%s' is not valid text sentiment.",
                     documentSentiment.getSentiment())));
         }
-        final SentimentConfidenceScorePerLabel confidenceScorePerLabel = documentSentiment.getDocumentScores();
+
+        final com.azure.ai.textanalytics.implementation.models.SentimentConfidenceScorePerLabel
```

Comment:
We do this because there is another type with this name, right? is it the generated one?
what is the difference between both?

Reply:
Yes. There is one generated one. Auto-generated one has setter methods.

Reply:
and why we can't use that one?

Reply:
You don't want to explore the setter to users and it is bad to change auto-generated code. It could be removed when doing code-gene with some settings. Created an issue to regenerate code base: https://github.com/Azure/azure-sdk-for-java/issues/8344